### DISCUSSION TITLE
Projects that are using CliFx 🚀

### DISCUSSION THREAD

**Discussion by willianantunes** [Tyrrrz/CliFx#108] [General]
There are many ways to find projects that are using CliFx, but one that I've been missing is actually something in this very repository. So that's why I created this discussion, so we can expose the projects that have been built, either by ourselves or that are being used internally, so we can tell lessons learned.

I would like to start with this playground project which is a Tic Tac Toe game through REST API:

- https://github.com/willianantunes/tic-tac-toe-csharp-playground/

I'm used to developing using Django. CliFx works pretty much like [Django Management Commands](https://docs.djangoproject.com/en/3.2/howto/custom-management-commands/), and it brings a lot of capabilities. You can apply migrations or even seed your database with fake data before running it (see this situation [here](https://github.com/willianantunes/tutorials/blob/ccf0270ddc93f057aab2b6c14d340672d2e86126/2021/XX/ef-core-how-to-handle-migrations-in-production/scripts/start-web-development.sh#L13-L14)), and it's pretty straightforward to test it as well. You can have it as a wrapper in one project to start your WEB API or your WORKER (worker sample [here](https://github.com/rafaelpadovezi/bookmark-manager/blob/28fc49f0bfcff9b5e827672f8e3f5e26d9fa0924/src/Commands/BookmarkInsertedConsumerCommand.cs#L18-L53)). Well, I think I could help you in any way by sharing a bit of some usage and the project above.

**Comment by Tyrrrz** [comment]
Thanks @willianantunes! That looks cool. I also remembered I started [this wiki page](https://github.com/Tyrrrz/CliFx/wiki/Projects-using-CliFx) but never actually updated it. Added your project there 🙂
